docs/plugins/autoslab.rst

```diff
@@ -0,0 +1,20 @@
+autoslab
+========
+
+.. dfhack-tool::
+    :summary: Automatically engrave slabs for ghostly citizens!
+    :tags: untested fort auto workorders
+    :no-command:
+
+Automatically queue orders to engrave slabs of existing ghosts. Will only queue
+an order if there is no existing slab with that unit's memorial engraved and
+there is not already an existing work order to engrave a slab for that unit
```

Comment:
Maybe note that the player is still responsible for both crafting the base unengraved slab and also for placing the engraved slabs as a building in-game.

Reply:
if they're not already there, we might want to add slabs to the default library manager orders in the `orders` library. they might be in `rockstock`, and we might move them to `basic`?

Reply:
> For the record I haven't _tested_ the changes on linux

DF only has a windows build, so testing on Linux (via wine/proton) isn't any better than testing on Windows. In general, though, DF does not have behavior differences on different platforms, and we don't ask contributors to test on anything other than what they normally run.